Doppler.Sap/Utils/SapSystemHelper.cs

```diff
@@ -0,0 +1,22 @@
+using Doppler.Sap.Enums;
+
+namespace Doppler.Sap.Utils
+{
+    public class SapSystemHelper
+    {
+        public static string GetSapSystemByBillingSystem(int billingSystem)
+        {
+            switch ((ResponsabileBillingEnum)billingSystem)
+            {
+                case ResponsabileBillingEnum.GBBISIDE:
+                case ResponsabileBillingEnum.Mercadopago:
+                    return "AR";
+                case ResponsabileBillingEnum.QuickBookUSA:
+                case ResponsabileBillingEnum.QBL:
+                    return "US";
+                default:
+                    return "";
+            }
```

Comment:
Zazzz, con esto perdimos el dinamismo que teníamos :P

Reply:
What do you think about adding a new setting for each system that is BillingSystemsToSupport and based on that we get the corresponding one?

Reply:
For example: 
```
"AR": {
        .....
        "BillingSystemsToSupport": "9,13"
        .....
}
```

Reply:
It could be, or the next step could configure the mapping between `ResponsabileBilling` with `SapSystem` in a configuration:

appsettings:

```json
// . . .
  "SapSystemByResponsabileBilling": {
    "GBBISIDE": "AR",
    "Mercadopago": "AR",
    "QuickBookUSA": "US",
    "QBL": "US"
  },
// . . .
```

Reply:
good, I will create a new task for it.

Reply:
For it I created the [task](https://makingsense.atlassian.net/jira/software/projects/DAT/boards/62/backlog?selectedIssue=DAT-215) in the backlog.